pkg/kube/inject/inject.go

```diff
@@ -695,15 +699,28 @@ func IntoObject(injector Injector, sidecarTemplate Templates, valuesConfig strin
 	if name == "" {
 		name = deploymentMetadata.Name
 	}
+	namespace := metadata.Namespace
+	if namespace == "" {
+		namespace = defaultNamespace
```

Comment:
This is not a safe assumption for kubeinject IMO. I think people will run `istioctl kube-inject -f <file-without-namespace> | kubectl apply -f - -n my-namespace`. This would get set to 'default' here which doesn't seem right?

Reply:
Thanks for the point

Reply:
Removed the `default` value, and only print the namespace if it is set in the yaml config.